core/core_modules/creator.js

```diff
@@ -1,20 +1,22 @@
 exports.match = function(event, commandPrefix) {
-	return event.body === commandPrefix + 'creator';
+        return event.body === commandPrefix + 'creator';
 };
 
 exports.run = function(api, event) {
-	var creators = [
-		'Thank you to:',
-		'- Matthew Knox my awesome creator about whom nobody is allowed to insult.',
-		'- Dion Woolley for my slackiness and his random, often broken contributions.',
-		'- Jay Harris for my fawltyness and profoundness of being.',
-		'- James Fairbairn for veing va vampire.',
-		'- and the other weird people who contributed to me.'
-	];
-	api.sendMessage(creators.join('\n'), event.thread_id);
-	return false;
+        var creators = [
+                'Thank you to:',
+                '- Matthew Knox my awesome creator about whom nobody is allowed to insult.',
+                '- Dion Woolley for my slackiness and his random, often broken contributions.',
+                '- Jay Harris for my fawltyness and profoundness of being.',
+                '- James Fairbairn for veing va vampire.',
+                '- and the other weird people who contributed to me.'
+        ];
+        api.sendMessage(creators.join('\n'), event.thread_id);
+        return false;
 };
 
 exports.help = function(commandPrefix) {
-	return [[commandPrefix + 'creator','Lists contributors to this platform', 'Does not list all contributors only a few of the people that helped build me when I was still finding my feet.\n And you should insult Matthew whenever you want,\nor maybe not,\nno pls,\nstap,\n......\n......\nA wild Marvin appears']];
+        return [[commandPrefix + 'creator', 'Lists contributors to this platform',
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 space characters but found 8.](https://www.codacy.com/app/theprojectshop/Kassy/file/2589031841/issues/source?bid=3282987&fileBranchId=3291254#l19)